### ISSUE TITLE
Installation script gives wrong directions

### ISSUE THREAD

**Issue by tjtuom** [Compass/compass#186]
When installing compass with this command

```
compass init rails . -r susy -u susy    
```

it told me to add these lines to my layout

```
%head
  = stylesheet_link_tag 'compiled/screen.scss', :media => 'screen, projection'
  = stylesheet_link_tag 'compiled/print.scss', :media => 'print'
  = stylesheet_link_tag 'compiled/ie.scss', :media => 'screen, projection'
```

It should of course say .css and not .scss.

**Comment by chriseppstein** [comment]
what? your browser doesn't understand scss? lame.

**Comment by mirisuzanne** [comment]
hmmm, this is one way to approach graceful degradation. 

We only support future browsers. Modern browsers are so 2010 and will be obsolete in no time. Future browsers are the browsers of the future. Try to keep up.

**Comment by chriseppstein** [comment]
Fix error in installation instructions on scss-based projects.
Closed by 45ca5eb1470088553cea30181ed2a66092593cbd.
